src/ontology/edits/oeo-physical.omn

```diff
@@ -2666,6 +2712,18 @@ Class: SolarEnergy
         <http://purl.obolibrary.org/obo/RO_0000091> some <http://openenergy-platform.org/ontology/oeo-physical/RenewableFuel>
     
     
+Class: SolarPark
```

Comment:
We should rename this class. Field photovoltaic powerplant?

Reply:
renamed